### PR TITLE
Pt pt translation

### PR DESCRIPTION
**Pull request by zecarlos1957**
## Purpose
Portuguese translation to[ WIN32] Module [SETUPAPI/SHELL32]

### CODE REVIEW

File: dll/cpl/sysdm/lang/pt-PT.rc [hunk 1]
```diff
@@ -0,0 +1,355 @@
+/* Portuguese Language resource file
+ *
+ * Traduzido por: Jose Carlos Jesus 20-06-2019 zecarlos1957@hotmail.com
+ */
+
+
+
+
```
Comment (Fraizeraust) [inline dll/cpl/sysdm/lang/pt-PT.rc]: Please remove unnecessary white spaces.

File: dll/win32/setupapi/lang/pt-BR.rc [hunk 2]
```diff
@@ -13,46 +13,32 @@ BEGIN
...
-    LTEXT "", SOURCESTRORD, 7, 18, 194, 11, WS_CHILD | WS_VISIBLE | WS_GROUP
-    LTEXT "Destino:", -1, 7, 30, 77, 11, WS_CHILD | WS_VISIBLE | WS_GROUP
-    LTEXT "", DESTSTRORD, 7, 41, 194, 22, WS_CHILD | WS_VISIBLE | WS_GROUP
-    CONTROL "", PROGRESSORD, "setupx_progress", WS_CHILD | WS_VISIBLE | WS_TABSTOP, 7, 63, 194, 13
-END
 
 IDPROMPTFORDISK DIALOG 0, 0, 260, 120
 STYLE DS_MODALFRAME | WS_CAPTION | WS_SYSMENU
-CAPTION "Files Needed"
+CAPTION "Arquivos necess�rios"
```
Comment (Fraizeraust) [inline dll/win32/setupapi/lang/pt-BR.rc]: You've got to fix the broken characters. As an advice, use Notepad++ as it doesn't break the encoding on resource files.

File: dll/win32/setupapi/lang/pt-PT.rc [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    LTEXT "Origem:", -1, 7, 7, 77, 11, WS_CHILD | WS_VISIBLE | WS_GROUP
+    LTEXT "", SOURCESTRORD, 7, 18, 194, 11, WS_CHILD | WS_VISIBLE | WS_GROUP
+    LTEXT "Destino:", -1, 7, 30, 77, 11, WS_CHILD | WS_VISIBLE | WS_GROUP
+    LTEXT "", DESTSTRORD, 7, 41, 194, 22, WS_CHILD | WS_VISIBLE | WS_GROUP
+    CONTROL "", PROGRESSORD, "setupx_progress", WS_CHILD | WS_VISIBLE | WS_TABSTOP, 7, 63, 194, 13
+END
+
+IDPROMPTFORDISK DIALOG 0, 0, 260, 120
+STYLE DS_MODALFRAME | WS_CAPTION | WS_SYSMENU
+CAPTION "Arquivos necess�rios"
```
Comment (Fraizeraust) [inline dll/win32/setupapi/lang/pt-PT.rc]: You forgot to fix the encoding here (as well as on the lines below).

Comment (zecarlos1957) [reply dll/win32/setupapi/lang/pt-PT.rc]: it will be this time

File: dll/win32/samsrv/lang/pt-PT.rc [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    IDS_ALIAS_POWER_USERS_NAME "Utilizadores avançados"
+    IDS_ALIAS_POWER_USERS_COMMENT "Utilizadores avançados têm a maioria dos direitos administrativos com algumas restrições."
+    IDS_ALIAS_USERS_NAME "Utilizadores"
+    IDS_ALIAS_USERS_COMMENT "Os utilizadores são impedidos de fazer alterações em todo o sistema."
+    /*
+     * ATTENTION: 
+     *   If you translate the administrator account name, keep IDS_USER_ADMINISTRATOR_NAME and
+     *   syssetup.dll:IDS_ADMINISTRATOR_NAME synchronized.
+     */
+    IDS_USER_ADMINISTRATOR_NAME "Administrador"
```
Comment (Fraizeraust) [inline dll/win32/samsrv/lang/pt-PT.rc]: Please check that `IDS_ADMINISTRATOR_NAME` string resource from `syssetup` Portuguese file is also translated.

Comment (zecarlos1957) [reply dll/win32/samsrv/lang/pt-PT.rc]: As it says in the comment on line 18 I did the translation in the two files as it says.
Do you want me to remove the translation of that line in both files?

Comment (zecarlos1957) [reply dll/win32/samsrv/lang/pt-PT.rc]: OK buddy. is done and passed the test.

**Pull request merged**